services/kafka/Dockerfile

```diff
@@ -32,27 +25,20 @@ ENV KAFKA_RELEASE_ARCHIVE kafka_${KAFKA_SCALA_VERSION}-${KAFKA_RELEASE}.tgz
 WORKDIR /tmp
 RUN wget --no-verbose http://www.us.apache.org/dist/kafka/${KAFKA_RELEASE}/${KAFKA_RELEASE_ARCHIVE} && \
     wget --no-verbose https://dist.apache.org/repos/dist/release/kafka/${KAFKA_RELEASE}/${KAFKA_RELEASE_ARCHIVE}.md5  && \
-    npm install -g express@4.8.0 nano@5.10.0 process@0.10.1 websocket@1.0.17 && \
   echo VERIFY CHECKSUM: && \
   gpg --print-md MD5 ${KAFKA_RELEASE_ARCHIVE} 2>/dev/null && \
   cat ${KAFKA_RELEASE_ARCHIVE}.md5 && \
   tar -zx -C /kafka --strip-components=1 -f ${KAFKA_RELEASE_ARCHIVE} && \
   rm -rf kafka_*
 
-#
-# Install js app dependencies
-ADD monitor  /monitor
-RUN cd /monitor && npm install
-
-RUN cd /kafka/libs/ && wget --no-verbose https://jcenter.bintray.com/org/slf4j/slf4j-log4j12/1.7.6/slf4j-log4j12-1.7.6.jar
 ADD config /kafka/config
 ADD start.sh /start.sh
 
 # Set up a user to run Kafka
 RUN chmod uog+rx /start.sh && \
   groupadd kafka && \
   useradd -d /kafka -g kafka -s /bin/false kafka && \
-  chown -R kafka:kafka /kafka /data /logs /monitor
+  chown -R kafka:kafka /kafka /data /logs
```

Comment:
Nope, this `/logs` is used by Kafka itself. See https://hub.docker.com/r/ches/kafka/~/dockerfile/ (which is the Dockerfile which i believe was used as the base for ours and is the one I replaced ours in #1395 )